Neos.Cache/Classes/Backend/FileBackend.php

```diff
@@ -281,7 +281,11 @@ public function flush(): void
     {
         Files::emptyDirectoryRecursively($this->cacheDirectory);
         if ($this->frozen === true) {
-            @unlink($this->cacheDirectory . 'FrozenCache.data');
+            try {
+                @unlink($this->cacheDirectory . 'FrozenCache.data');
+            } catch (\Throwable $e) {
+                // Double-shrug catching
```

Comment:
Thanks for the explanation, so what about:
```suggestion
                // 🤷🏻‍♀️🤷🏼‍♂️
```
haha. Just kidding, I like your suggestion!